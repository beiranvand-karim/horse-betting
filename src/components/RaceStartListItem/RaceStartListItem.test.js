/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {RaceStartListItem} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"
import game from "../../backend/game"

describe('<RaceStartListItem/>', () => {
   const start = game.races[0].starts[0];

   it('should render without crashing', () => {
      shallow(<RaceStartListItem start={start}/>)
   });

   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><RaceStartListItem start={start}/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});
