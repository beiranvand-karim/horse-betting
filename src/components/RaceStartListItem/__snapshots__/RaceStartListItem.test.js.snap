// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RaceStartListItem/> should matches the snapshot 1`] = `
<div>
  <h5>
    start number: 
    1
  </h5>
  <h5>
    horse name: 
    Lani Kai Beach
  </h5>
  <h5>
    driver first name: 
    J N
     
    Douglass
  </h5>
  <h5>
    horse father: 
    Camlach
  </h5>
</div>
`;
